src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/ContinueSetupForSingleApprenticeshipByProvider.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@using SFA.DAS.EmployerAccounts.Web.Extensions
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.CallToActionViewModel
```

Comment:
Please pass in just the ApprenticeshipViewModel rather than the CohortViewModel

Reply:
done